packages/cli/src/utils/logger.ts

```diff
@@ -0,0 +1,7 @@
+export const logger = {
+  ...console,
+
+  log: (...args: any[]) => {
+    process.env.DISCOVERY_DEBUG === 'true' ? console.log(...args) : undefined
+  },
+}
```

Comment:
type-safe

```suggestion
export const logger = new Proxy(console, {
  get(target, prop: keyof Console) {
    if (prop === 'log') {
      return (...args: any[]) => {
        if (process.env.DISCOVERY_DEBUG === 'true') {
          target.log(...args);
        }
      };
    }
    return target[prop];
  }
});
```

Reply:
@emersonlaurentino thank you so much for the `Proxy` knowledge 🫶